### PR TITLE
Add term picker for choosing between 1-year and 2-year plans

### PR DESCRIPTION
**Pull request by adamziel**
This is an implementation of the following feature:

zrzut ekranu 2018-03-22 o 10 29 33

Test plan (on both desktop and mobile):

1. Pick any plan either as a first plan or as an upgrade plan
2. Go to checkout
3. Click on the another term
4. Confirm that cart was updated
5. I don't think API is ready for actually finalising the purchase so testing ends here

### CODE REVIEW

File: client/components/term-picker-option/index.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+		const { term, translate } = this.props;
+		switch ( term ) {
+			case TERM_BIENNIALLY:
+				return translate( '2 years' );
+
+			case TERM_ANNUALLY:
+				return translate( '1 year' );
+
+			case TERM_MONTHLY:
+				return translate( '1 month' );
```
Comment (a8ci18n) [inline client/components/term-picker-option/index.jsx]: Hi! I've found a possible matching string that has already been translated **77** times:
`translate( 'month' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/2-year-plans-step3-checkout-subscription-length-component)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/term-picker-option/index.jsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+	};
+
+	getTermText() {
+		const { term, translate } = this.props;
+		switch ( term ) {
+			case TERM_BIENNIALLY:
+				return translate( '2 years' );
+
+			case TERM_ANNUALLY:
+				return translate( '1 year' );
```
Comment (a8ci18n) [inline client/components/term-picker-option/index.jsx]: Hi! I've found a possible matching string that has already been translated **71** times:
`translate( 'year' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/term-picker-option/index.jsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+				value: this.props.value,
+			} );
+		}
+	};
+
+	getTermText() {
+		const { term, translate } = this.props;
+		switch ( term ) {
+			case TERM_BIENNIALLY:
+				return translate( '2 years' );
```
Comment (a8ci18n) [inline client/components/term-picker-option/index.jsx]: Hi! I've found a possible matching string that has already been translated **59** times:
`translate( 'Years' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/2-year-plans-step3-checkout-subscription-length-component)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/components/term-picker-option/index.jsx (hunk 1 above)
Comment (akirk) [reply client/components/term-picker-option/index.jsx]: Please change this to `translate( '%s month', '%s months', { count: 1, args: '1' } );`.

File: client/components/term-picker-option/index.jsx (hunk 2 above)
Comment (akirk) [reply client/components/term-picker-option/index.jsx]: Please change this to `translate( '%s year', '%s years', { count: 1, args: '1' } );`.

File: client/components/term-picker-option/index.jsx (hunk 3 above)
Comment (akirk) [reply client/components/term-picker-option/index.jsx]: Please change this to `translate( '%s year', '%s years', { count: 2, args: '2' } );`.

**Pull request closed without merging**